app/assets/stylesheets/pages/_report_form.css.scss

```diff
@@ -23,5 +25,21 @@
     align-items: center;
     display: flex;
     flex-direction: column;
-}
 
+    input{
+      align-self: flex-end;
+      background: $placeholder-bg-color;
+      border-radius: 5px;
+      border: none;
+      box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+      cursor: pointer;
+      height: 26px;
+      left: 495px;
+      padding-left: 5px;
+      top: 390px;
+      width: 264px;
+    }
+    input:hover{
+      background-color: $placeholder-bg-color-hover;
+    }
+}
```

Comment:
add an space between your class name an your curly braces, and add a blank line between each blocks